src/lib/system.ts

```diff
@@ -24,3 +24,4 @@ export const isCloud = MODE === Mode.CLOUD;
 export const isSelfHosted = MODE !== Mode.CLOUD;
 export const hasStripePublicKey = !!VARS.STRIPE_PUBLIC_KEY;
 export const GRACE_PERIOD_OVERRIDE = false;
+export const TRIAL_PERIOD_OVERRIDE = true;
```

Comment:
This is an override to disable trials? It's a little confusing..I thought it should have been false to disable trials 🧐 . Would it be clearer to name it `DISABLE_TRIAL_PERIOD`?

Reply:
If anything, maybe add a comment? Anyways, you're free to merge whenever.